src/ws_tray.js

```diff
@@ -44,8 +43,8 @@ function WSTray() {
 }
 
 function setNotifyIcon() {
-    if (is.osx()) {
-        if (osxPrefs.isDarkMode()) {
+    if (is.macOS()) {
+        if (tray.remote.systemPreferences.isDarkMode()) {
```

Comment:
did this ever work? 🤔 

![image](https://user-images.githubusercontent.com/5712253/68375713-9173be00-0104-11ea-85b4-a1575382a69a.png)

Created https://github.com/chef/chef-workstation-app/issues/122